src/components/step/step.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Container, Title, SubTitle, StepButton } from './styles';
+
+interface IDivProp {
+    title: string;
+    subTitle: string;
+    onClick: () => void;
+  }
+  
+const Div = ({ title, subTitle, onClick }: IDivProp) => {
+    return(
+     <Container> 
+            <div>
+                <Title>{title}</Title>
+            </div>
+            <div>
+                <SubTitle>{subTitle}</SubTitle>
+            </div>
+            <div style={{ margin: 17 }}>
```

Comment:
Remove extra block for margin, lets add it in style, like margin top or bottom what you need. . .